File: FizzBuzz Solutions/fizzbuzz.ml
let fizzbuzz n =
  for i = 1 to n do
    if i mod 15 = 0 then
      print_endline "FizzBuzz"
    else if i mod 3 = 0 then
      print_endline "Fizz"
    else if i mod 5 = 0 then
      print_endline "Buzz"
    else
      print_endline (string_of_int i)
  done;;

fizzbuzz 100;;

File: FizzBuzz Solutions/FizzBuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        int i;

        for (i = 0; i < 100; i++) {
            if (((i + 1) % 3 == 0) && ((i + 1) % 5 == 0)) {
                System.out.println("FizzBuzz");
            }
            else if ((i + 1) % 3 == 0) {
                System.out.println("Fizz");
            }
            else if ((i + 1) % 5 == 0) {
                System.out.println("Buzz");
            }
            else {
                System.out.println(i+1);
            }
        }
    }
}